pkg/controller/servicebindingrequest/servicebindingrequest_controller.go

```diff
@@ -97,5 +100,47 @@ func (r *ReconcileServiceBindingRequest) Reconcile(request reconcile.Request) (r
 		return reconcile.Result{}, err
 	}
 
-	return reconcile.Result{}, nil
+	nn := types.NamespacedName{Namespace: instance.Spec.CSVNamespace,
+		Name: instance.Spec.BackingOperatorName}
+	csv := &olmv1alpha1.ClusterServiceVersion{}
+	err = r.client.Get(context.TODO(), nn, csv)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	evList := []corev1.EnvVar{}
+
+	for _, crd := range csv.Spec.CustomResourceDefinitions.Owned {
+		for _, spec := range crd.SpecDescriptors {
+			pt := spec.Path
+			for _, xd := range spec.XDescriptors {
+				if strings.HasPrefix(xd, "urn:alm:descriptor:servicebinding:secret:") {
```

Comment:
Preferably it should  be servicebindingrequest instead of servicebinding

Reply:
Done. 414d93276e54cf11e225c11efbc514dc9dd067ac